mlflow/java/client/pom.xml

```diff
@@ -45,15 +45,6 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
-    <dependency>
```

Comment:
I removed these deps as they were a bit much to annotate 4 params. We can add back as needed, but they would need to be shaded for all customers.